edelphi/src/main/java/fi/metatavu/edelphi/jsons/resources/ImportGDocsJSONRequestController.java

```diff
@@ -11,23 +11,40 @@
 import fi.metatavu.edelphi.dao.resources.FolderDAO;
 import fi.metatavu.edelphi.dao.resources.GoogleDocumentDAO;
 import fi.metatavu.edelphi.dao.resources.GoogleImageDAO;
+import fi.metatavu.edelphi.dao.resources.ResourceDAO;
 import fi.metatavu.edelphi.dao.users.UserDAO;
-import fi.metatavu.edelphi.domainmodel.actions.DelfoiActionScope;
+import fi.metatavu.edelphi.domainmodel.base.Delfoi;
+import fi.metatavu.edelphi.domainmodel.panels.Panel;
 import fi.metatavu.edelphi.domainmodel.resources.Folder;
+import fi.metatavu.edelphi.domainmodel.resources.Resource;
 import fi.metatavu.edelphi.domainmodel.users.User;
 import fi.metatavu.edelphi.i18n.Messages;
 import fi.metatavu.edelphi.jsons.JSONController;
 import fi.metatavu.edelphi.smvcj.AccessDeniedException;
+import fi.metatavu.edelphi.smvcj.LoginRequiredException;
+import fi.metatavu.edelphi.smvcj.PageNotFoundException;
 import fi.metatavu.edelphi.smvcj.SmvcRuntimeException;
 import fi.metatavu.edelphi.smvcj.controllers.JSONRequestContext;
+import fi.metatavu.edelphi.smvcj.controllers.RequestContext;
 import fi.metatavu.edelphi.utils.GoogleDriveUtils;
 import fi.metatavu.edelphi.utils.ResourceUtils;
 
 public class ImportGDocsJSONRequestController extends JSONController {
 
-  public ImportGDocsJSONRequestController() {
-    super();
-    setAccessAction(DelfoiActionName.MANAGE_PANEL_MATERIALS, DelfoiActionScope.PANEL);
+  @Override
+  public void authorize(RequestContext requestContext) throws LoginRequiredException, AccessDeniedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)